src/app/common/hooks/balance/use-balances.tsx

```diff
@@ -82,14 +81,8 @@ export function useBalances({ btcAddress, stxAddress }: UseBalanceArgs) {
         (isPendingBtcBalance && Boolean(btcAddress)),
       totalBalance,
       availableBalance,
-      availableUsdBalance: i18nFormatCurrency(
-        availableBalance,
-        availableBalance.amount.isGreaterThanOrEqualTo(highBalance.amount) ? 0 : 2
-      ),
-      totalUsdBalance: i18nFormatCurrency(
-        totalBalance,
-        totalBalance.amount.isGreaterThanOrEqualTo(highBalance.amount) ? 0 : 2
-      ),
+      availableUsdBalance: formatCurrency(availableBalance),
```

Comment:
I added a note into [LEA-3091](https://linear.app/leather-io/issue/LEA-3091/implement-balance-services-in-extension) for implementing the balances service in the `extension` as we should take time to also refactor our hardcoded `fiat` / `usd` strings in favour of using `baseCurrency` and `quoteCurrency` as we do in `mobile`.